pkg/engine/armvariables.go

```diff
@@ -110,8 +115,8 @@ func getK8sMasterVars(cs *api.ContainerService) (map[string]interface{}, error)
+			kubernetesVersion, isHostedMaster, isIPv6DualStackFeatureEnabled, azureStackCNIParams),
```

Comment:
Do we need comma as azureStackCNIParams has added comma. I assume you have tested the scenario where this value is empty.

Reply:
The unit test in armvariables_test.go tests how this string is generated. I don't add a comma in the original string since we don't want a trailing comma if the insertion string is empty. The insertion string starts with a comma when it isn't empty.